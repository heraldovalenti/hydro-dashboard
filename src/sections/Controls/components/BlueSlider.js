import { Slider } from '@mui/material';
import { withStyles } from '@mui/styles';
import COLORS from '../../../types/colors';

export const BlueSlider = withStyles({
  root: {
    color: COLORS.TURQUOISE.BASE,
    height: 8,
  },
})(Slider);
