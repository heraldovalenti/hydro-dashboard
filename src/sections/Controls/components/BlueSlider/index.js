import React from 'react';
import { withStyles } from '@material-ui/core/styles';
import COLORS from '../../../../types/colors';
import { Slider } from '@material-ui/core';

const BlueSlider = withStyles({
  root: {
    color: COLORS.TURQUOISE.BASE,
    height: 8,
  },
})(Slider);

export default BlueSlider;
