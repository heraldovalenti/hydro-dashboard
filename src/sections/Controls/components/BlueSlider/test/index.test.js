import React from 'react';
import BlueSlider from '../';
import ShallowRenderer from 'react-test-renderer/shallow';

describe('<BlueSlider/>', () => {
  it('default', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<BlueSlider />);
    const tree = renderer.getRenderOutput();
    expect(tree).toMatchSnapshot();
  });
});
