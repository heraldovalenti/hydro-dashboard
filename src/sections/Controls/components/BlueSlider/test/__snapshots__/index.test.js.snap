// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BlueSlider/> default 1`] = `
<ForwardRef(WithStyles)
  classes={
    Object {
      "root": "WithStyles(ForwardRef(Slider))-root-1",
    }
  }
/>
`;
